src/ApiService/ApiService/host.json

```diff
@@ -3,8 +3,8 @@
     "logging": {
         "applicationInsights": {
             "samplingSettings": {
-                "isEnabled": false,
-                "excludedTypes": "Request;Trace;Dependency;Event;Exception"
+                "isEnabled": true,
+                "excludedTypes": "Request"
```

Comment:
I think these were changed recently by Noah to improve our sampling issues, does this PR negate the need for it in the first place?

Reply:
if you turn this off <- then there will be no logs at all

Reply:
in the previous implementation this was "extra" logs that were coming from "lower layers". In this implementation this controls all of logs